lib/tasks/generate_test_data.rake

```diff
@@ -5,5 +5,7 @@ end
 
 desc 'Generate test applications for existing providers'
 task generate_test_applications: :environment do
-  GenerateTestApplications.new.perform
+  Sidekiq::Testing.inline! do
```

Comment:
Should this be in the rake task?

Reply:
I've moved it inside the `GenerateTestApplications` worker. I think this means that it will apply to provider initiated test data generation as well as the rake task.